docs/design/keyboard-shortcuts.md

```diff
@@ -42,244 +60,180 @@ Immediately *below* (not inside) the **\<VersionOverrides\>** element in the man
+Users of your add-in can reassign the actions of the add-in to alternate keyboard combinations.
 
-If the user is logged into Office, the custom combinations are saved in the user's roaming settings per platform. Customizing shortcuts are currently not supported for anonymous users.
+Use the [Office.actions.replaceShortcuts](/javascript/api/office/office.actions#office-office-actions-replaceshortcuts-member) method to assign a user's custom keyboard combinations to your add-ins actions. The method takes a parameter of type `{[actionId:string]: string|null}`, where the `actionId`s are a subset of the action IDs that must be defined in the add-in's extended manifest JSON. The values are the user's preferred key combinations. The value can also be `null`, which will remove any customization for that `actionId` and revert to the specified default keyboard combination.
+
+If the user is logged into Microsoft 365, the custom combinations are saved in the user's roaming settings per platform. Customizing shortcuts aren't currently supported for anonymous users.
 
 ```javascript
 const userCustomShortcuts = {
-    SHOWTASKPANE:"CTRL+SHIFT+1", 
-    HIDETASKPANE:"CTRL+SHIFT+2"
+    ShowTaskpane: "Ctrl+Shift+1",
+    HideTaskpane: "Ctrl+Shift+2"
 };
+
 Office.actions.replaceShortcuts(userCustomShortcuts)
-    .then(function () {
-        console.log("Successfully registered.");
+    .then(() => {
+        console.log("Successfully registered shortcut.");
     })
-    .catch(function (ex) {
-        if (ex.code == "InvalidOperation") {
-            console.log("ActionId does not exist or shortcut combination is invalid.");
+    .catch((error) => {
+        if (error.code == "InvalidOperation") {
+            console.log("ActionId doesn't exist or shortcut combination is invalid.");
         }
     });
 ```
 
-To find out what shortcuts are already in use for the user, call the [Office.actions.getShortcuts](/javascript/api/office/office.actions#office-office-actions-getshortcuts-member) method. This method returns an object of type `[actionId:string]:string|null}`, where the values represent the current keyboard combination the user must use to invoke the specified action. The values can come from three different sources:
+To find out what shortcuts are already in use for the user, call the [Office.actions.getShortcuts](/javascript/api/office/office.actions#office-office-actions-getshortcuts-member) method. This method returns an object of type `[actionId:string]:string|null}`, where the values represent the current keyboard combination the user must use to invoke the specified action. The values can come from three different sources.
 
 - If there was a conflict with the shortcut and the user has chosen to use a different action (either native or another add-in) for that keyboard combination, the value returned will be `null` since the shortcut has been overridden and there is no keyboard combination the user can currently use to invoke that add-in action.
```

Comment:
```suggestion
- If there was a conflict with the shortcut and the user has chosen to use a different action (either native or another add-in) for that keyboard combination, the value returned will be `null` since the shortcut has been overridden and there's no keyboard combination the user can currently use to invoke that add-in action.
```